NN_chatCmd.js

```diff
@@ -99,33 +111,56 @@ nerthus.chatCmd.map["map"] = function(ch)
 {
     var map_url = ch.t.split(" ").slice(1).join(" ");
     map_url = nerthus.chatCmd.extractUrlFromDecorator(map_url);
-    $("#ground").css("backgroundImage","url(" + map_url + ")")
+    var map_container = $("#_ng-map-" + nerthus.chatCmd.currentMap);
+    if (map_container.length)
+        map_container.css("backgroundImage", "url(" + map_url + ")");
+    else
+    {
+        $("<img id='_ng-map-" + nerthus.chatCmd.currentMap + "'>")
+        .css({
+            "backgroundImage": "url(" + map_url + ")",
+            "width": map.x << 5,
```

Comment:
serio <<? zamień to na normalne mnożenie

Reply:
<< jest szybsze :/ Ale dobra